App.js

```diff
@@ -1,16 +1,102 @@
 import { StatusBar } from 'expo-status-bar';
-import React from 'react';
-import { StyleSheet, View } from 'react-native';
+import React, {useState} from 'react';
```

Comment:
I see so has app.js become kinda like the functional blood/neurosystem of the application ?  Where the views are like a index of the components filled with that app blood.